packages/requisitions/src/RequestRequisition/DetailView/Footer/Footer.tsx

```diff
@@ -7,35 +7,21 @@ import {
 } from '@openmsupply-client/common';
 import React, { FC } from 'react';
 import {
-  getRequestRequisitionTranslator,
+  getRequisitionTranslator,
   requestRequisitionStatuses,
 } from '../../../utils';
 import { RequestRequisitionFragment, useRequestRequisition } from '../../api';
 import { StatusChangeButton } from './StatusChangeButton';
 
 export const createStatusLog = (requisition: RequestRequisitionFragment) => {
-  const statusIdx = requestRequisitionStatuses.findIndex(
-    s => requisition.status === s
-  );
   const statusLog: Record<RequisitionNodeStatus, null | undefined | string> = {
-    [RequisitionNodeStatus.Draft]: null,
-    [RequisitionNodeStatus.Sent]: null,
-    [RequisitionNodeStatus.Finalised]: null,
-
+    [RequisitionNodeStatus.Draft]: requisition.createdDatetime,
+    [RequisitionNodeStatus.Sent]: requisition.sentDatetime,
+    [RequisitionNodeStatus.Finalised]: requisition.finalisedDatetime,
     // Keeping typescript happy, not used for request requisitions.
     [RequisitionNodeStatus.New]: null,
   };
 
-  if (statusIdx >= 0) {
-    statusLog[RequisitionNodeStatus.Draft] = requisition.createdDatetime;
-  }
-  if (statusIdx >= 1) {
-    statusLog[RequisitionNodeStatus.Sent] = requisition.sentDatetime;
-  }
-  if (statusIdx >= 2) {
-    statusLog[RequisitionNodeStatus.Finalised] = requisition.finalisedDatetime;
-  }
```

Comment:
This was a bit funny because of the mock server and making sure the dates were set properly. In reality the dates would just be `null` anyway, so we don't need to do all this